scripts/make_UKB_study_table.py

```diff
@@ -79,6 +83,30 @@ def to_int_safe(i):
         return None
 
 
+def make_trait_reported_string(s_raw):
+    '''Takes the raw trait name and outputs transformed name'''
+
+    # Replace any double spaces with single
+    s_raw = re.sub(r' +', r' ', s_raw)
+
+    # Assert no "|" in trait name
+    assert "|" not in s_raw
```

Comment:
Assertion is good, but I think there should be some report here:
```suggestion
    assert "|" not in s_raw, f"Reported trait ({s_raw}), contains invalid character."
```